cmd/spiffe-helper/config/config.go

```diff
@@ -24,6 +30,10 @@ type Config struct {
 	CmdArgsDeprecated                  string `hcl:"cmdArgs"`
 	CertDir                            string `hcl:"cert_dir"`
 	CertDirDeprecated                  string `hcl:"certDir"`
+	CertFileMode                       string `hcl:"cert_file_mode"`
+	KeyFileMode                        string `hcl:"key_file_mode"`
+	JwtBundleFileMode                  string `hcl:"jwt_bundle_file_mode"`
+	JwtSvidFileMode                    string `hcl:"jwt_svid_file_mode"`
```

Comment:
can we type these as uint64?

Reply:
Should be uint32 I think?

Reply:
Not working. It says
```
root.cert_file_mode: unknown kind to decode into: uint32
```

Reply:
The same thing happens if I use `uint64`. It says

```
root.cert_file_mode: unknown kind to decode into: uint64
```

Reply:
Changed it back to a string.

Reply:
It looks like with hcl version 1.0.0, the only types [supported](https://github.com/hashicorp/hcl/blob/v1.0.0/decoder.go#L89-L114) are

* Boo
* Float32 / Float64
* Int32 / Int64
* Interface:
* Map
* Pointer
* Slice
* String
* Struct

So we'd have to choose a signed rather than unsigned integer for this.

Reply:
I looked into hclsimple in hcl v2. But it only accepts files with an hcl extension. I had trouble working out how we could upgrade to v2 of the hcl dependency. However, it's worth noting no other spiffe project uses v2, so maybe that's a path we shouldn't go down anyway.

Reply:
Yes i think hcl v2 is out of scope.

Reply:
How about using a sign int then? Its closer than a string.

Reply:
Changed to int.